scalafix-core/src/main/scala-3/scalafix/util/Compat.scala

```diff
@@ -0,0 +1,9 @@
+package scalafix.util
+
+import scala.collection.immutable.IndexedSeq
+
+object Compat {
+  type View[T] = collection.SeqView[T] //, IndexedSeq[T]]
+  type SeqView[T] = collection.SeqView[T] //, IndexedSeq[T]]
```

Comment:
if I provide `[T], IndexedSeq[T]]` it fails compiling saying there are too many type args and it only expects one [A] type arg.. would you know how to fix this?

Reply:
Thanks @armanbilge, now it's more clear